test/web-platform-tests/to-run.yaml

```diff
@@ -472,9 +472,7 @@ form_owner_and_table_3.html: [fail, complex form owner semantics not implemented
 
 DIR: html/semantics/forms/resetting-a-form
 
-reset-event.html: [fail, Unknown]
-reset-form-event-realm.html: [fail, Unknown]
-reset-form.html: [fail, Unknown]
+reset-event.html: [fail, Expects event.isTrusted to equal true]
```

Comment:
This can be fixed by using the internal `_dispatch()` method which allows setting `isTrusted`, instead of `dispatchEvent()`.

Reply:
Oh awesome! I did not realize that was the problem. Thank you! This has been fixed as well.